circuit_maintenance_parser/parsers/aws.py

```diff
@@ -60,31 +76,67 @@ def parse_text(self, text):
             This maintenance is scheduled to avoid disrupting redundant connections at =
             the same time.
         """
-        data = {"circuits": []}
+        text_map = {
+            "^Account ?I?D?: ([0-9]+)": "account",
+            "^Start Time: ([A-Z][a-z]{2}, [0-9]{1,2} [A-Z][a-z]{2,9} [0-9]{4} [0-9]{2}:[0-9]{2}:[0-9]{2} [A-Z]{2,3})": "start",
+            "^End Time: ([A-Z][a-z]{2}, [0-9]{1,2} [A-Z][a-z]{2,9} [0-9]{4} [0-9]{2}:[0-9]{2}:[0-9]{2} [A-Z]{2,3})": "end",
+            "(?<=from )([A-Z][a-z]{2}, [0-9]{1,2} [A-Z][a-z]{2,9} [0-9]{4} [0-9]{2}:[0-9]{2}:[0-9]{2} [A-Z]{2,3}) to ([A-Z][a-z]{2}, [0-9]{1,2} [A-Z][a-z]{2,9} [0-9]{4} [0-9]{2}:[0-9]{2}:[0-9]{2} [A-Z]{2,3})": "start_and_end",
+        }
+
+        regex_keys = re.compile("|".join(text_map), re.IGNORECASE)
+
+        data = {"circuits": [], "start": "", "end": ""}
```

Comment:
again, I prefer to not start with empty "start" and "end" values, if we can't solve them, let's leave them empty